templates/docker-compose.template

```diff
@@ -24,35 +24,32 @@ services:
   packet-forwarder:
     image: nebraltd/hm-pktfwd:{{PKTFWD_VERSION}}
     depends_on:
-      - helium-miner
+      - gatewayrs
     restart: always
     privileged: true
     volumes:
       - pktfwdr:/var/pktfwd
     environment:
       - FIRMWARE_VERSION={{FIRMWARE_VERSION}}
+      - FIRMWARE_SHORT_HASH={{ENV.FIRMWARE_SHORT_HASH}}
 
-  helium-miner:
-    image: nebraltd/hm-gatewayrs:{{GATEWAY_VERSION}}
-    expose:
-      - "1680"
-      - "4467"
+  gatewayrs:
```

Comment:
as mentioned on previous PR we have to have this as helium-miner

Reply:
@shawaj Due to your previous comment I've dived deep into packet-forwarder and couldn't find any reference regarding to the "helium-miner" name obligation. Right now it still depends to gatewayrs, no problems there. Could you point me to the right direction, so I can find the helium-miner name dependency.

Reply:
it is referenced in hm-pktfwdr and hm-pyhelper

Reply:
Done!